pyqtgraph/parametertree/parameterTypes/basetypes.py

```diff
@@ -319,24 +319,37 @@ def __init__(self, param, depth):
         self.optsChanged(self.param, self.param.opts)
 
     def updateDepth(self, depth):
-        ## Change item's appearance based on its depth in the tree
-        ## This allows highest-level groups to be displayed more prominently.
-        if depth == 0:
-            for c in [0,1]:
-                self.setBackground(c, QtGui.QBrush(QtGui.QColor(100,100,100)))
-                self.setForeground(c, QtGui.QBrush(QtGui.QColor(220,220,255)))
-                font = self.font(c)
-                font.setBold(True)
+        """
+        Change item's appearance based on its depth in the tree
+        This allows highest-level groups to be displayed more prominently.
+        """
+        app = QtWidgets.QApplication.instance()
+        palette = app.palette()
+        textColor = palette.text().color()
+        backgroundColor = palette.button().color().darker(110)
+        altBackgroundColor = palette.dark().color()
+
+        # if stylesheet and 'QDarkStyleSheet' in stylesheet:
+        if app.styleSheet(): # Assume the stylesheet is qdarkstyle
```

Comment:
Thanks for checking for a stylesheet --I recommend looking for "qdarkstyle" explicitly in the sheet, rather than just checking it is nonzero in length. The header of qdarkstyle's sheet is

```
/* ---------------------------------------------------------------------------
    WARNING! File created programmatically. All changes made in this file will be lost!
    Created by the qtsass compiler v0.3.0
    The definitions are in the "qdarkstyle.qss._styles.scss" module
--------------------------------------------------------------------------- */
/* Light Style - QDarkStyleSheet ------------------------------------------ */
/*
See Qt documentation:
```

So there are a few magic strings you can check for

Reply:
With load_stylesheet(palette=LightPalette), it looks a bit wonky:
![image](https://user-images.githubusercontent.com/23620506/142133329-7e485afa-c22a-4507-ba40-3f99c6cfc325.png)

Note that "DarkStyle" is commented out in this case as opposed to Light Style

Reply:
![image](https://user-images.githubusercontent.com/5187375/142156777-e69fec87-49ea-4444-904a-f7d6cd003c95.png)

Hows this? I feel like the text could be better on the outermost groups.. maybe not quite so dark or something with color.